docs/source/index.rst

```diff
@@ -85,6 +87,8 @@ For this example, we use some data loaded from disk and query it with a SQL comm
          result.sum.mean().compute()
 
 
+..
+    TODO - Incorrectly formatted
 .. toctree::
```

Comment:
This isn't being formatted properly (see https://github.com/dask-contrib/dask-sql/blob/932f7067131c4b829aa85d2790b7a79f62dd9910/docs/source/index.rst).